src/components/Company/index.jsx

```diff
@@ -0,0 +1,15 @@
+
+const Company = ({logo, jobRole, companyName}) => {
+
+    return (
+    <div className={'container mt-2 ms-1'}>
```

Comment:
I don't think any of these classNames need to be inside curly brackets at all, as they aren't interpreted javascript, just strings. Same for width and height below, it can all be just strings without curly brackets.